src/main/assisted-installer-controller/assisted_installer_main.go

```diff
@@ -165,20 +187,28 @@ func main() {
 		logger.Infof("Finished all")
 	}()
 
-	hasValidvSphereCredentials := common.HasValidvSphereCredentials(mainContext, client, logger)
-	if hasValidvSphereCredentials {
-		logger.Infof("Has valid vSphere credentials: %v", hasValidvSphereCredentials)
+	if cluster != nil && cluster.Platform != nil {
+		platformType = string(*cluster.Platform.Type)
+	}
+
+	hasValidvSphereCredentials := false
+	if platformType == string(models.PlatformTypeVsphere) {
```

Comment:
We have this validation https://github.com/openshift/assisted-installer/pull/896/files#diff-a5bd908493da50da6a2905424525321bb4b7b81b90da798952ae8aac3ec53546R258 ? 
Need both places?

Reply:
It is not need in both places. Thanks for spotting it.